### ISSUE TITLE
Move frontend starter to client-boilerplate

### ISSUE THREAD

**Issue by austinstarin** [punkave/frontend-starter#53]

**Comment by abea** [comment]
Wasn't the idea that this would be applicable for non-apostrophe projects as well?

**Comment by austinstarin** [comment]
Yes, but the effort to get this into the boilerplate is too high. If we need it for another project I would rather remove it from the boilerplate for that use case.
